CMakeLists.txt

```diff
@@ -15,13 +15,26 @@ set(NUM_CHUNKS 1)
 set(NUM_THREADS 160)
 set(NUM_VECTOR_LANES 1)
 
-if(CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM")
-        set(CMAKE_CXX_FLAGS "-fp-model=precise")
-elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Intel")
-        message("-- Detected Intel classic compiler which will be deprecated soon.")
-        message("-- It is recommended you use IntelLLVM compiler.")
-        set(CMAKE_CXX_FLAGS "-diag-disable=10441 -fp-model=precise")
-endif()
+if (SPECFEMPP_ENABLE_PYTHON)
```

Comment:
Can you define the `SPECFEMPP_ENABLE_PYTHON` option `OFF` above in the this file?